src/main/java/com/bb3/bodybuddybe/users/UsersBlockEnum.java

```diff
@@ -0,0 +1,5 @@
+package com.bb3.bodybuddybe.users;
+
+public enum UsersBlockEnum {
+    허가, 휴먼, 차단
```

Comment:
상수 이름은 영어로 작성하고 한글 명칭을 추가적으로 사용하는게 enum을 관리하기에 편리하지 않을까요?

```suggestion
   ACTIVE("허가"),
   INACTIVE("휴면"),
   BLOCKED("차단");
   
   private String krName;
    
   UsersBlockEnum(String krName) { 
       this.krName = krName;
   } 
```

Reply:
오.. 너무 좋은 것 같습니다!